docs/development/minimal-reproductions.md

```diff
@@ -1,49 +1,80 @@
 # About minimal reproductions
 
-You may be asked to provide a "minimal reproduction" repository to help the Renovate team debug bug reports or work on a feature.
+We may ask you to create a "minimal reproduction" repository to help us fix bugs or work on a feature.
 
-This document explains why we need a minimal reproduction, why using large repositories to debug is undesirable, and how you can create a good minimal reproduction.
+This document explains why we need a minimal reproduction, why we won't use your production repository to debug, and how to create a good minimal reproduction.
+
+## Help yourself by creating a minimal reproduction
+
+A minimal reproduction helps us to:
+
+- Fix your bug faster
+- Work on your feature request faster
+- Confirm the cause of the behavior
+
+It's fastest if you - as the bug reporter or feature requester - create the reproduction.
+
+## How the Renovate developers use your minimal reproduction
+
+TODO: `@rarkins` should fill out this section.
```

Comment:
@rarkins Can you please suggest a text that explains how the Renovate developers use the minimal reproduction?